### PR TITLE
.NET 5 Preview 8 APIs

### PR DESCRIPTION
**Pull request by BillWagner**
CI build for .NET 5 Preview 8

### CODE REVIEW

File: xml/FrameworksIndex/net-5.0.xml [hunk 1]
```diff
@@ -4414,51 +4384,6 @@
       <Member Id="M:System.Guid.TryParseExact(System.String,System.String,System.Guid@)" />
       <Member Id="M:System.Guid.TryWriteBytes(System.Span{System.Byte})" />
     </Type>
-    <Type Name="System.Half" Id="T:System.Half">
```
Comment (mairaw) [inline xml/FrameworksIndex/net-5.0.xml:4417]: is Half really being dropped?

Comment (BillWagner) [reply xml/FrameworksIndex/net-5.0.xml:4417]: ping @carlossanlop @safern

Comment (carlossanlop) [reply xml/FrameworksIndex/net-5.0.xml:4417]: I don't think that is correct.

Comment (tannergooding) [reply xml/FrameworksIndex/net-5.0.xml:4417]: This definitely shouldn't be getting dropped.

It is notably still in the ref assembly: <https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime/ref/System.Runtime.cs#L2206>
nor was it flagged in the API diff: <https://github.com/dotnet/core/commit/851bb0cfc828a64ba63531f471a51f7c1d9e96ff>

Comment (safern) [reply xml/FrameworksIndex/net-5.0.xml:4417]: I'll take a look at what's wrong.

Comment (safern) [reply xml/FrameworksIndex/net-5.0.xml:4417]: Good catch, for some reason I got the Preview5 ref assemblies, I don't know what happened. Will send the new ones now.

Comment (safern) [reply xml/FrameworksIndex/net-5.0.xml:4417]: I've sent the updated ref assemblies.

Comment (BillWagner) [reply xml/FrameworksIndex/net-5.0.xml:4417]: I'll close this PR, and start a new CI build.

**Pull request closed without merging**